src/components/fields/date-field/date-field.tsx

```diff
@@ -141,7 +152,10 @@ export const DateField = (props: IGenericFieldProps<IDateFieldSchema>) => {
 			notFutureRule?.["notFuture"] && LocalDate.now(),
 			withinDaysRange?.endDate,
 		]);
-		const disabledDatesProps = excludedDatesRule?.["excludedDates"];
+		const disabledDatesProps = [
+			...(excludedDatesRule?.["excludedDates"] || []),
+			...DateTimeHelper.calculateDisabledBeyondDaysDates({ ...beyondDaysRule?.["beyondDays"], dateFormat }),
```

Comment:
should only calculate if `beyondDaysRule` is set, otherwise today is disabled by default even for conditional rule